src/main/cli/cli.c

```diff
@@ -4829,6 +4829,10 @@ if (buildKey) {
     cliPrintLinefeed();
 #endif // USE_GPS
 
+#ifdef LOCALE
+    cliPrintLinef("%s %s", STR_LOCALE_SETUP, STR_LOCALE);
```

Comment:
cli is not translated ...

Reply:
No..., but are the way to inform ex configurator about chosen language.

Reply:
but you have a point regarding #ifdef LOCALE, I remove